haas/ext/obm/haas_obm.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2015 Massachusetts Open Cloud Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS
+# IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+""" An obm driver for recursive HaaS.
+It invokes Base HaaS to execute various OBM actions on nodes 
+"""
+
+from sqlalchemy import Column, String, Integer, ForeignKey
+
+from haas.model import Obm
+
+
+class Haas_obm(Obm):
```

Comment:
Are we doing recursive HaaS with this submit, or just OBM drivertization? If the latter, then this should be removed.